pallets/mt/src/mock.rs

```diff
@@ -21,9 +23,9 @@ frame_support::construct_runtime!(
 		UncheckedExtrinsic = UncheckedExtrinsic,
 	{
 		System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
-		HasherPallet: pallet_hasher::{Pallet, Call, Storage, Event<T>},
 		Balances: pallet_balances::{Pallet, Call, Storage, Event<T>},
-		SMT: pallet_smt::{Pallet, Call, Storage, Event<T>},
+		HasherPallet: pallet_hasher::{Pallet, Call, Storage, Event<T>},
+		Smt: pallet_smt::{Pallet, Call, Storage, Event<T>},
```

Comment:
Better as pallet_mt and MT as this doesn't support sparse insertions yet.

Reply:
So you suggest renaming this to just `MerkleTree`?

Reply:
Yea or MT, your judgement is good